openslides/core/projector.py

```diff
@@ -139,3 +139,10 @@ class Message(ProjectorElement):
     def check_data(self):
         if self.config_entry.get('message') is None:
             raise ProjectorException('No message given.')
+
+
+class ListOfSpeakersOverlay(ProjectorElement):
+    """
+    List of speakers (last, current, next) on the projector. Rendered as overlay.
+    """
+    name = 'core/speakeroverlay'
```

Comment:
Can't test this; can_see_projector gets too much permissions 
(I hesitate to open yet another issue when I may just have forgotten something)

---**details:**---
I did:
```
git checkout master
rm -r personal-data/
pip install -r requirements.txt
npm install
cleared browser cache
restarted browser
log in as admin
create group "onlyprojector(6)"  "permissions": ["core.can_see_projector"]
create user "John", groups_id: [6]
create some agenda and motions
log out
delete cache again
log in as John
```
I can see what I should not see: the agenda, each agenda_item, motions/categories, 
motions in detail, all workflows, all assignments in detail